README.md

```diff
@@ -1,10 +1,17 @@
 # Kafka protocol Erlang library
 
-Code generated from BNF definitions https://cwiki.apache.org/confluence/display/KAFKA/A+Guide+To+The+Kafka+Protocol
+This is a kafka wire format encode/decode library, not a kafka client.
+See https://github.com/klarna/brod for kafka client implementation.
+
+Code generated from org.apache.kafka.common.protocol.Protocol.
+
+## How to generate kafka.bnf
+Ensure you have java (1.7+) and gradle (2.0+) installed.
+Change kafka version in priv/kafka_protocol_bnf/build.gradle if needed.
 
+  (cd priv/kafka_protocol_bnf && gradle -q run) > priv/kafka.bnf
```

Comment:
that was my intent. Though, I'd rather not run this for every build, only when file's missing.